tests/ui/methods/no-auto-borrow-vec-to-slice.stderr

```diff
@@ -1,53 +1,53 @@
 error[E0599]: no method named `test_mut` found for struct `Vec<{integer}>` in the current scope
-  --> $DIR/auto-ref-slice-plus-ref.rs:7:7
+  --> $DIR/no-auto-borrow-vec-to-slice.rs:6:7
```

Comment:
Suggestion: actually, the old test name is arguably better, the test actually exercises autoderef *and* autoref, maybe call this instead `vec-autoderef-autoref`